vpr/src/draw/draw.cpp

```diff
@@ -1448,17 +1448,17 @@ static void draw_rr_chan(int inode, const ezgl::color color, ezgl::renderer* g)
     int coord_min = -1;
     int coord_max = -1;
     if (type == CHANX) {
-        coord_min = device_ctx.rr_nodes[inode].xlow();
-        coord_max = device_ctx.rr_nodes[inode].xhigh();
+        coord_min = rr_graph.node_xlow(RRNodeId(inode));
```

Comment:
Could we declare inode to be or type RRNodeId and get rid of a bunch of these casts?